frontend/src/SearchMenu/SearchMenu.tsx

```diff
@@ -134,6 +137,9 @@ function SearchMenu(props: ISearchMenuProps) {
               key={mural.id}
               handleMuralClick={props.handleMuralClick}
               handleCancel={props.handleCancel}
+              setSelectedMural={props.setSelectedMural}
+              setSelectedTour={props.setSelectedTour}
+              setSelectedCollection={props.setSelectedCollection}
```

Comment:
in another world, we would generalize some of this functionality to `setSelectedResource` then differentiate by `resourceType` at the App.tsx level or something

Reply:
That’s a great idea haha I’ll modify it in my PR! Thanks for pointing it out